apps/documentation/src/app/components/example/example.html

```diff
@@ -44,11 +44,11 @@
 
   <div class="relative mt-10 w-full flex-1">
     @if (mode() === 'preview') {
-      <div
-        class="flex h-fit min-h-[280px] items-center justify-center rounded-xl border border-zinc-200 bg-zinc-50 p-8 dark:border-zinc-800 dark:bg-zinc-950"
-      >
-        <ng-container [ngComponentOutlet]="component" />
-      </div>
+    <div
+      class="flex h-fit min-h-[280px] items-center justify-center rounded-xl border border-zinc-200 bg-zinc-50 p-8 dark:border-zinc-800 dark:bg-zinc-950"
+    >
+      <ng-container [ngComponentOutlet]="component" />
+    </div>
```

Comment:
I immediately thought of this project when I saw the style guide update 😂 Now we officially don't have a way to identify the external Angular templates from the file name, with inline templates being the only way to do so. Classic early adopter issues!

Your generator worked btw! I'm planning to revert the change for now as a bunch of files are going to change yet again just for random prettier changes, and my team is going to adopt inline templates going forward.

Reply:
😂 I know, always a risk you take being an early adopter, but I can't help myself 😂!

That's great it worked! Yeah, I'm coming round to the idea of inline templates more and more myself